### ISSUE TITLE
LocalPlayer a nil value when trying to write version info.

### ISSUE THREAD

**Issue by lzinga** [lzinga/TTTWeightedTraitorSelection#13]
@KajTorvaldGrey messaged me with this error on Steam. This is for when the system checks for a new version here on github.

Link to location https://github.com/lzinga/TTTWeightedTraitorSelection/blob/master/tttweightsystem/lua/autorun/weightsystem_autorun.lua#L115

```
[ERROR] lua/autorun/weightsystem_autorun.lua:115: attempt to call global 'LocalPlayer' (a nil value)
1. onsuccess - lua/autorun/weightsystem_autorun.lua:115
2. unknown - lua/includes/modules/http.lua:35
```

Last time I was working on this addon it was working, so maybe something changed with LocalPlayer method or just need to change it so it has a nil check or loop over all players in the game and write it instead of using LocalPlayer.

**Comment by LaurenceKaye** [comment]
I fixed this in my fork and never pull requested it for some reason. I'll clean it up and request it.

**Comment by LaurenceKaye** [comment]
Fixed by #14
